internal/monitors/logstash/logstash/config.go

```diff
@@ -0,0 +1,82 @@
+package logstash
+
+import (
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/signalfx-agent/internal/core/config"
+)
+
+// Config for this monitor
+type Config struct {
+	config.MonitorConfig `yaml:",inline" acceptsEndpoints:"true" singleInstance:"false"`
+	// The hostname of Logstash monitoring API
+	Host string `yaml:"host" default:"127.0.0.1"`
+	// The port number of Logstash monitoring API
+	Port uint16 `yaml:"port" default:"9600"`
+	// If true, the agent will connect to the host using HTTPS instead of plain HTTP.
+	UseHTTPS bool `yaml:"useHTTPS"`
+	// The maximum amount of time to wait for API requests
+	TimeoutSeconds int `yaml:"timeoutSeconds" default:"5"`
+	// Whether it will send all extra hot threads metrics as well.
+	EnableExtraHotThreadsMetrics bool `yaml:"enableExtraHotThreadsMetrics"`
```

Comment:
The groups functionality covers all of these boolean flags and they are unnecessary.